spec/ics-002-client-semantics/README.md

```diff
@@ -366,6 +368,7 @@ type verifyChannelState = (
 type verifyPacketData = (
   clientState: ClientState,
   height: Height,
+  delayPeriod: uint64,
```

Comment:
These require the `currentTimestamp` as well

Reply:
This is the SDK interface, but isn't it only that way because we don't pass a `sdk.Context`?